src/chromatix/functional/propagation.py

```diff
@@ -115,21 +127,28 @@ def exact_propagate(
         z: A float that defines the distance to propagate.
         n: A float that defines the refractive index of the medium.
         N_pad: A keyword argument integer defining the pad length for the
-        propagation FFT (NOTE: should not be a Jax array, otherwise a
-        ConcretizationError will arise when traced!).
+            propagation FFT (NOTE: should not be a Jax array, otherwise a
+            ConcretizationError will arise when traced!). Use padding calculator
+            utilities from ``chromatix.functional.propagation`` to calculate the
+            padding.
+        cval: The value for the padding, 0 by default
+        kykx: If provided, defines the orientation of the propagation. Should be an
+            array of shape `[2,]` in the format [ky, kx].
     """
     # Calculating propagator
+    if kykx is None:
+        kykx = jnp.zeros(2)
```

Comment:
Default should be zeros instead of. It should also be `jnp.zeros((2, ))`.

Reply:
done